app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The like should also belong to an `Author` with a class name of `User` not a user, kindly fix this.
e.g
`belongs_to :author, class_name: 'User'`